### PR TITLE
Update docs for new "default-impl-name" property for load-flow and nettwork

### PR DESCRIPTION
**Pull request by jonenst**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
NO

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Docs

### CODE REVIEW

File: docs/configuration/modules/index.md [hunk 1]
```diff
@@ -81,3 +81,4 @@ module2:
 - [security-analysis](security-analysis.md)
 - [simulation-parameters](simulation-parameters.md)
 - [table-formatter](table-formatter.md)
+- [network](network.md)
```
Comment (MioRtia) [inline docs/configuration/modules/index.md]: These links are alphabetically ordered, could you please do the same? It makes searching for specific modules easier and allows us to check quickly if the page is updated when a new module is added.

File: docs/configuration/modules/network.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+title: network
+layout: default
+---
+
+The `network` module is used to configure the network default implementation name. The network implementation is the set of classes implementing all the network elements, such as VoltageLevel or Generator.
```
[the remark this answers is not in the document]
Comment (jonenst) [reply docs/configuration/modules/network.md]: Not sure if we should document them yet. They are not yet usable through the defaut-impl-name mechanism (for example they don't even have a name yet).

File: docs/configuration/modules/index.md (hunk 1 above)
Comment (jonenst) [reply docs/configuration/modules/index.md]: done

File: docs/configuration/modules/network.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jonenst) [reply docs/configuration/modules/network.md]: I added a mention of the classic in-memory implementation named "Default"

**Pull request merged**